### PR TITLE
Renamed the Program name field to title

### PR DESCRIPTION
**Pull request by clintonb**
This is more inline with how we handle courses and course runs.

Note that usage of programs will be temporarily broken between the time it takes to run the migrations and deploy the code. This is acceptable since programs are not yet being used in production.

### CODE REVIEW

**Comment by clintonb** [comment]
@bderusha @Dillon-Dumesnil please review

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -397,8 +397,9 @@ class Program(TimeStampedModel):
         verbose_name=_('UUID')
     )
 
-    name = models.CharField(
-        help_text=_('The user-facing display name for this Program.'),
+    # TODO Rename this field to title
```
Comment (bderusha) [inline course_discovery/apps/course_metadata/models.py]: Todo == done?

**Comment by bderusha** [comment]
👍

File: course_discovery/apps/course_metadata/models.py (hunk 1 above)
Comment (clintonb) [inline course_discovery/apps/course_metadata/models.py]: Haha! Yep!

**Comment by Dillon-Dumesnil** [comment]
👍

**Pull request merged**